### PR TITLE
[DCM-KIT] Q&A Section #585

### PR DESCRIPTION
**Pull request by ClosedSourcerer**
## Description

This PR contains DCM-Kit content for release 24.08.

The following Issues are addressed: 
|Issue ID|Name|Structure|Content|
|--|--|--|--|
|[585](https://github.com/eclipse-tractusx/sig-release/issues/585)|[DCM-KIT] Q&A Section||X|

The structure for the Q&A is pulled via https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/981

(https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/981 AND https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/984 AND https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/985) REPLACE https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/969 

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [ ] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [X] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by MWADCM — COMMENTED** [review]
OK

**Review by LukasSchmetz — APPROVED** [review]
LGTM @ClosedSourcerer

**Review by jschu1 — APPROVED** [review]

File: docs-kits/kits/DCM-Kit/adoption-view/qna.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+|2|Where can I read about DCM?|<https://catenax-ev.github.io/docs/next/standards/CX-0128-DemandandCapacityManagementDataExchange> contains the released standards. The DCM standard has the ID CX-0128.|
+|3|Are there translations of the DCM standard?|There are no official translations. Only the original standard, written in English, is normative.|
+|4|Is there a standard “easy to read” DCM presentation available?|No, however the standard contains an abstract, which provides a management overview. The KIT itself is receiving regular updates.|
+|5|What is the benefit of using DCM within the Catena-X DataSpace instead of using my individual own process?| Adopting DCM, as defined by Catena-X, has the unique benefit of eliminating the need for defining process and technical interfaces individually with every one of your business partners. This allows you to execute demand and capacity management with every single one of your business partners in an identical manner, greatly reducing overhead.|
```
Comment (danielmiehle) [inline docs-kits/kits/DCM-Kit/adoption-view/qna.md:17]: General Typo: DataSpace --> Data Space

File: docs-kits/kits/DCM-Kit/adoption-view/qna.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+|8|Do I need to be the part of Catena-X if I only want to use DCM?|Yes, joining the Catena-X dataspace (not the Catena-X Association) via an operating company is a prerequisite for participating in any Catena-X usecase.|
+|9|Is Cofinity-X required for DCM? Or are there other possibilities?| Currently, Cofinity-X is the only operating company of the Catena-X dataspace and therefore registration at Cofinity-X is a prerequisite for data exchange.|
+|10|Where can I get Catena-X related consultancy services|You can find Catena-X related consultancy services within the Cofinity-X [service marketplace](https://www.cofinity-x.com/service-marketplace/).|
```
Comment (danielmiehle) [inline docs-kits/kits/DCM-Kit/adoption-view/qna.md:22]: @jSchuetz88 @stephanbcbauer : Are there also list of certified solution provider / apps, advisory provider in the CX e.V.? I would rather use the neutral website such as the C-X e.V. website? Please advise!

**Review by danielmiehle — CHANGES_REQUESTED** [review]
Great Q/A section! Please check change request.

File: docs-kits/kits/DCM-Kit/adoption-view/qna.md (hunk 2 above)
Comment (jSchuetz88) [reply docs-kits/kits/DCM-Kit/adoption-view/qna.md:22]: As far as I know, we currently do not provide a list of qualified advisory services (I will cross-check). One option, however, would be the following link: https://catena-x.net/en/catena-x-introduce-implement/certification

File: docs-kits/kits/DCM-Kit/adoption-view/qna.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+|5|I participate in another Catena-X use case, will it simplify my access to the DCM use case and maybe even reduce my onboarding costs?|Yes, it will. Nearly all Catena-X use cases share a common infrastructure. Having an EDC in place and being already registered with an operating company,having assigned BPNL and BPNS, as well as a working wallet will reduce the on-boarding time and cost for DCM|
+|6|How often are Catena-X standards and in particular the DCM standard updated?|Catena-X standards can be updated on a quarterly basis, with breaking changes being allowed once a year. The DCM expert group is committed to introducing new features to the DCM standard without incurring breaking changes.|
```
Comment (jSchuetz88) [inline docs-kits/kits/DCM-Kit/adoption-view/qna.md:37]: We should add to the answer, that we intend to assure a backward compatibility for the two newest major releases.

File: docs-kits/kits/DCM-Kit/adoption-view/qna.md (hunk 1 above)
Comment (ClosedSourcerer) [reply docs-kits/kits/DCM-Kit/adoption-view/qna.md]: dataspace vs. data space is an age old debate und we consider IBM to be correct.
Nevertheless capitilization required correction. 

Fixed capatilization in commit: https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/984/commits/904beb151d8dd1cae567df8eca9202a09345c9a9

File: docs-kits/kits/DCM-Kit/adoption-view/qna.md (hunk 3 above)
Comment (ClosedSourcerer) [reply docs-kits/kits/DCM-Kit/adoption-view/qna.md]: Agreed.
Fixed via commit https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/984/commits/9c81d5998a4f2eb817f7e5cfcc5115cf825f7aaf

File: docs-kits/kits/DCM-Kit/adoption-view/qna.md (hunk 2 above)
Comment (jSchuetz88) [reply docs-kits/kits/DCM-Kit/adoption-view/qna.md:22]: This link could be references: https://catena-x.net/en/catena-x-introduce-implement/certification
The last figure provides a brief overview of the current qualified advisory services

**Review by jSchuetz88 — APPROVED** [review]
Thank you for including the feedback, looks good to me.

File: docs-kits/kits/DCM-Kit/adoption-view/qna.md (hunk 2 above)
Comment (ClosedSourcerer) [reply docs-kits/kits/DCM-Kit/adoption-view/qna.md:22]: > This link could be referenced: https://catena-x.net/en/catena-x-introduce-implement/certification Unfortunately, the overview is not comprehensive, but the last figure provides a general overview.

I do not think, that the suggested certification link works within the context and intend of the quoted Q&A section.
If the association, at some point, keeps a record of consulting companies, I will be more than happy to include that. But for now, the cofinity-x references need to remain.

**Review by maximilianong — APPROVED** [review]

**Comment by jSchuetz88** [comment]
@danielmiehle could you re-review?

**Review by danielmiehle — APPROVED** [review]

**Review by nhaenis — APPROVED** [review]

**Review by danielmiehle — APPROVED** [review]

**Review by maximilianong — APPROVED** [review]

**Pull request merged**